README.md

```diff
@@ -170,10 +152,19 @@ senate <- ggplot(us_senate, aes(x, y, colour = party_long)) +
        subtitle = "The party that has control of the Senate is encircled in black.") +
   scale_colour_manual(values = us_senate$colour,
                       limits = us_senate$party_long)
+```
+
+```
+## Error in loadNamespace(name): there is no package called 'ggrepel'
+```
+
+```r
 senate 
 ```
 
-![plot of chunk unnamed-chunk-7](figure/unnamed-chunk-7-1.png)
+```
+## Error in eval(expr, envir, enclos): object 'senate' not found
```

Comment:
should these (+line 156) be showing errors?

Reply:
Fixed - new laptop, hadn't installed `ggrepel`. Thanks.